sdk/trace/span.go

```diff
@@ -242,6 +244,7 @@ func (s *recordingSpan) SetAttributes(attributes ...attribute.KeyValue) {
 
 	// Otherwise, add without deduplication. When attributes are read they
 	// will be deduplicated, optimizing the operation.
+	s.attributes = slices.Grow(s.attributes, len(attributes))
```

Comment:
How about inlining the `slices.Grow` implementation instead of referencing `golang.org/x/exp`?

Reply:
Certainly happy to inline it if that's easier.

Reply:
Right now it is used only in `sdk/trace/span.go` right? I think we can add a comment and issue that we should use https://pkg.go.dev/slices#Grow instead once we stop supporting Go 1.20.

Reply:
Correct, after Go1.20 is no longer supported we can use slices#Grow instead. I've inlined as requested.